web/src/main/java/com/synopsys/integration/alert/web/api/config/ConfigActions.java

```diff
@@ -197,13 +197,13 @@ public ConfigActions(AuthorizationManager authorizationManager, DescriptorAccess
     protected ValidationActionResponse validateWithoutChecks(FieldModel resource) {
         if (!encryptionUtility.isInitialized() && !settingsDescriptorKey.getUniversalKey().equals(resource.getDescriptorName())) {
             ValidationResponseModel validationResponseModel = ValidationResponseModel.generalError(EncryptionSettingsValidator.ENCRYPTION_MISSING);
-            return new ValidationActionResponse(validationResponseModel);
+            return new ValidationActionResponse(HttpStatus.BAD_REQUEST, validationResponseModel);
```

Comment:
This might be changing behavior.

Reply:
Although I can see now that this is the bug fix. Maybe a 500 rather than 400? Since it's not a problem with the request, but with the state of the server?